test/utils/log-parser.test.ts

```
@@ -63,7 +63,7 @@ Status: FAIL
 		const result = await parseLogFile(logPath);
 		expect(result).not.toBeNull();
 		expect(result?.jobId).toBe("check_src");
-		expect(result?.adapterFailures[0].adapterName).toBe("check");
+		expect(result?.adapterFailures[0]!.adapterName).toBe("check");
```

Biome (2.3.13)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)